client/client-multi/client-c/src/kaa/collections/kaa_list.c

```diff
@@ -309,3 +309,17 @@ void kaa_list_sort(kaa_list_t *list, match_predicate pred)
     }
     list->tail = node;
 }
+
+uint32_t kaa_list_hash(kaa_list_t *list, list_node_hash pred)
```

Comment:
Just interesting, would it be a problem what other SDKs use signed types - int and long?